src/lib/styles/tools/forms/_inline-field-elements.scss

```diff
@@ -54,3 +57,31 @@
     }
   }
 }
+
+// 2.
+@mixin min-tap-target($type) {
+  $input-width: theme.$field-check-input-size;
+
+  @if ($type == 'toggle') {
+    $input-width: theme.$field-check-input-toggle-width;
+  }
+
+  $tap-target-offset: calc((#{$input-width} - #{theme.$field-check-tap-target-size}) / 2);
+
+  @include accessibility.min-tap-target($size: theme.$field-check-tap-target-size, $center: false);
+
+  min-height: theme.$field-check-tap-target-size;
```

Comment:
Consider removing this line in context of `FormLayout` to keep vertical field gaps reasonably sized.